Prefabs/CameraRig/TrackedAlias/SharedResources/Scripts/TrackedAliasFacade.cs

```diff
@@ -94,5 +87,26 @@ public class TrackedAliasFacade : MonoBehaviour
         /// Retreives all of the linked CameraRig Right Controller Velocity Trackers.
         /// </summary>
         public List<VelocityTracker> RightControllerVelocityTrackers => cameraRigs.Select(rig => rig.RightControllerVelocity).Where(value => value != null).ToList();
+
+        /// <summary>

```

Comment:
According to the conventions properties come **after** events.

Maybe put them into a `#region`, too?